### PR TITLE
Rebrand as @zazuko package

### PR DESCRIPTION
**Pull request by tpluscode**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -1,19 +1,27 @@
 name: CI
-on: push
+on: [push, pull_request]
 jobs:
+  lint:
+    runs-on: ubuntu-20.04
```
Comment (ludovicm67) [inline .github/workflows/ci.yaml]:
```suggestion
    runs-on: ubuntu-latest
```

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v3
+      - run: npm ci
+      - run: npm run lint
+
   test:
     runs-on: ubuntu-20.04
```
Comment (ludovicm67) [inline .github/workflows/ci.yaml]:
```suggestion
    runs-on: ubuntu-latest
```

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
     strategy:
       matrix:
         node:
-          - '12'
-          - '14'
+          - '20'
+          - '18'
           - '16'
```
Comment (ludovicm67) [inline .github/workflows/ci.yaml]: Node 16 is dead now, we can focus on the supported versions:

```suggestion
```

**Review by ludovicm67 — COMMENTED** [review]
So this mean the version number will be reset?

I did some suggestions about the CI.

**Comment by ludovicm67** [comment]
Also don't forget to update the README badges :)

**Comment by tpluscode** [comment]
> So this mean the version number will be reset?

Unless you think it would be better to bump the current version and make this `3.0`?

File: README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
-# rdf-utils-fs
+# @zazuko/rdf-utils-fs
 
-[![Build Status](https://travis-ci.org/rdf-ext/rdf-utils-fs.svg?branch=master)](https://travis-ci.org/rdf-ext/rdf-utils-dataset)
-[![npm version](https://badge.fury.io/js/rdf-utils-fs.svg)](https://badge.fury.io/js/rdf-utils-dataset)
+![Build Status](https://github.com/zazuko/rdf-utils-fs/actions/workflows/ci.yaml/badge.svg)
+[![npm version](https://badge.fury.io/js/@zazuko/rdf-utils-fs.svg)](https://badge.fury.io/js/@zazuko/rdf-utils-dataset)
```
Comment (ludovicm67) [inline README.md]:
```suggestion
[![npm version](https://badge.fury.io/js/@zazuko%2Frdf-utils-fs.svg)](https://badge.fury.io/js/@zazuko%2Frdf-utils-dataset)
```

File: README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
-# rdf-utils-fs
+# @zazuko/rdf-utils-fs
 
-[![Build Status](https://travis-ci.org/rdf-ext/rdf-utils-fs.svg?branch=master)](https://travis-ci.org/rdf-ext/rdf-utils-dataset)
-[![npm version](https://badge.fury.io/js/rdf-utils-fs.svg)](https://badge.fury.io/js/rdf-utils-dataset)
+![Build Status](https://github.com/zazuko/rdf-utils-fs/actions/workflows/ci.yaml/badge.svg)
```
Comment (ludovicm67) [inline README.md]: Let's use the `master` branch status:

```suggestion
![Build Status](https://github.com/zazuko/rdf-utils-fs/actions/workflows/ci.yaml/badge.svg?branch=master)
```

**Comment by ludovicm67** [comment]
> Unless you think it would be better to bump the current version and make this `3.0`?

Having it in the continuity of the original package version will help to know that this is the future of the original package, with new features, and so on.
Else it's losing that semantic ; but I don't have a strong opinion on this.

**Comment by tpluscode** [comment]
Ok, let's continue from the original version.

**Review by ludovicm67 — APPROVED** [review]

**Pull request merged**